config/swagger.js

```diff
@@ -109,6 +109,19 @@ function bootstrapSwagger(app, config, logger, scope, cb) {
 		// To be used in test cases or getting configuration runtime
 		app.swaggerRunner = runner;
 
+		// Managing all the queries which were not caught by previous middlewares.
+		app.use((req, res, next) => {
```

Comment:
Do all of our previous middlewares always return json?

Reply:
They do. There are two ways:
- call successfully caught by Swagger : the fitting `lisk_response_formatter` sets the content-type
- call unsuccessfully caught by SWGGERt: this patch sets the content-type